src/components/Switch/Switch.tsx

```diff
@@ -0,0 +1,80 @@
+﻿import { cloneElement, FC } from 'react';
+import classNames from 'classnames'
+import RcSwitch, { SwitchChangeEventHandler, SwitchClickEventHandler } from 'rc-switch'
+import styles from '@/components/Switch/Switch.module.scss'
+import "rc-switch/assets/index.css"
+
+export type SwitchProps = {
+  title?: string;
+  defaultChecked?: boolean;
+  checked?: boolean;
+  disabled?: boolean;
+  checkedChildren?: React.ReactNode;
+  unCheckedChildren?: React.ReactNode;
+  onChange?: SwitchChangeEventHandler;
+  onKeyDown?: React.KeyboardEventHandler<HTMLButtonElement>;
+  onClick?: SwitchClickEventHandler;
+  tabIndex?: number;
+  /**
+   * Default: vertical
+   */
+  type?: 'vertical' | 'horizontal',
+  className?: string;
+  labelClassName?: string;
+  loading?: boolean;
+  loadingIcon?: React.ReactElement;
+  style?: React.CSSProperties;
+  size?: 'small' | 'large';
+}
+
+const Switch: FC<SwitchProps> = ({
+  title,
+  defaultChecked,
+  checked,
+  disabled,
+  checkedChildren,
+  unCheckedChildren,
+  onChange,
+  onKeyDown,
+  onClick,
+  tabIndex,
+  type = 'vertical',
+  className,
+  labelClassName,
+  loadingIcon,
+  style,
+  size = 'large',
+  loading = false,
```

Comment:
We don't need all these props to be exposed. Take a look at the `src/admin-ui/src/components/Switch/index.js` in the Admin UI and only use these props.